src/services/object/object.class.ts

```diff
@@ -1,7 +1,7 @@
 import { Service, SequelizeServiceOptions } from 'feathers-sequelize'
 import { Application } from '../../declarations'
 
-export class XrObjects extends Service {
+export class Objects extends Service {
```

Comment:
Since all the other services are non-plural names now, maybe it makes sense to make this `XrObject`, to match? ("Object" is too general-purpose anyway, this is specifically an in-xr object).